index.bs

```diff
@@ -6545,14 +6545,14 @@ At this time, one [=credential property=] is defined: the [=resident key credent
+This [=client extension|client=] [=registration extension=] and [=authentication extension=] allows a [=[RP]=] to evaluate outputs from a pseudo-random function (PRF) associated with a [=credential=]. The PRFs provided by this extension map from {{BufferSource}}s of any length to 32-byte {{ArrayBuffer}}s.
 
 As a motivating example, PRF outputs could be used as symmetric keys to encrypt user data. Such encrypted data would be inaccessible without the ability to get assertions from the associated [=credential=]. By using the provision below to evaluate the PRF at two inputs in a single [=assertion=] operation, the encryption key could be periodically rotated during [=assertions=] by choosing a fresh, random input and reencrypting under the new output. If the evaluation inputs are unpredictable then even an attacker who could satisfy [=user verification=], and who had time-limited access to the authenticator, could not learn the encryption key without also knowing the correct PRF input.
 
 This extension is implemented on top of the [[FIDO-CTAP]] `hmac-secret` extension. It is a separate [=client extension=] because `hmac-secret` requires that inputs and outputs be encrypted in a manner that only the user agent can perform, and to provide separation between uses by WebAuthn and any uses by the underlying platform. This separation is achieved by hashing the provided PRF inputs with a context string to prevent evaluation of the PRFs for arbitrary inputs.
 
+The `hmac-secret` extension provides two PRFs per credential: one which is used for requests where [=user verification=] is performed and another for all other requests. This extension only exposes a single PRF per credential and, when implementing on top of `hmac-secret`, that PRF MUST be the one used for when [=user verification=] is performed. This overrides the {{UserVerificationRequirement}} if neccessary.
```

Comment:
I think this is a good idea to only use the UV version. Rather than the extension overriding the uv param, should the extension be ignored if UV wasn't returned / set?

Reply:
Discussed on the call of 2023-02-22:

When the uv and prf requests conflict, one of them has to win. My feeling is that prf should win because:

1. The RP may not be able to function without PRF results, but an extra UV is just an inconvenience for the user.
2. Platforms already upgrade UV automatically in a number of cases, such as discoverable-credential requests.

So I (and the group on the call) are still leaning towards giving priority to the prf request.